gst/tensor_decoder/tensordec_bounding_boxes_core.cc

```diff
@@ -0,0 +1,231 @@
+/**
+ * Copyright (C) 2018 Samsung Electronics Co., Ltd. All rights reserved.
```

Comment:
I don't see any reason to use C++ for this code.
Instead of ifstream (it seems you are using C++ for ifstream), glib may help you:
https://developer.gnome.org/glib/stable/glib-File-Utilities.html